src/utils/saml.ts

```diff
@@ -900,13 +889,28 @@ const assertionValidation = (
                                 new Error("Invalid AuthnContextClassRef value")
                             )
                           )
+                          .map(
+                            AuthnContextClassRef =>
+                              AuthnContextClassRef.textContent
+                          )
                           .chain(
                             fromPredicate(
                               AuthnContextClassRef => {
-                                return (
-                                  AuthnContextClassRef.textContent ===
-                                  requestAuthnContextClassRef
-                                );
+                                return requestAuthnContextClassRef ===
```

Comment:
since it looks like we are forcing a comparison for "minimum" here we should hardcode
`optionsRACComparison = "minimum"` into the strategy instead of accepting it as an option

Reply:
Forcing `RACComparison = "minimum"` imho will be less transparent and limits the library use context. We can instead extend the preValidator to support all possible comparison method (`exact, minimum, maximum, better`). Anyway, this code inside the preValidate never will throw an error with all the possible RACComparison's option provided (maybe only with `maximum` if the Response use RAC lower than the Request).